src/js/components/Quote.js

```diff
@@ -0,0 +1,52 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Box from './Box';
+import Paragraph from './Paragraph';
+import Props from '../utils/Props';
+
+const CLASS_ROOT = 'quote';
+
+export default class Quote extends Component {
+  render () {
+    let classes = classnames(
+      CLASS_ROOT,
+      this.props.className,
+      {
+        [`border-color-index-${this.props.borderColorIndex}`]: this.props.borderColorIndex,
+        [`${CLASS_ROOT}--${this.props.size}`]: this.props.size,
+        [`${CLASS_ROOT}--emphasize-quote`]: this.props.emphasizeQuote,
+        [`${CLASS_ROOT}--emphasize-credit`]: this.props.emphasizeCredit
+      }
+    );
+
+    let boxProps = Props.pick(this.props, Object.keys(Box.propTypes));
+
+    return (
+      <Box {...boxProps} className={classes}>
+        {this.props.quote.map((quote, index) =>
+          <Paragraph key={`quote-body-${index}`} size="large" className={`${CLASS_ROOT}__body`}>{quote}</Paragraph>
+        )}
+        <Paragraph margin="none" className={`${CLASS_ROOT}__credit`}>{this.props.quoteCredit}</Paragraph>
+      </Box>
+    );
+  }
+};
+
+Quote.propTypes = {
+  borderColorIndex: PropTypes.string,
+  size: PropTypes.oneOf(['small', 'medium', 'large', 'full']),
+  quote: PropTypes.array.isRequired,
```

Comment:
Quote is an array to allow for multi-paragraph quotes.  There is an example of this in the PR I submitted for grommet-docs (https://github.com/grommet/grommet-docs/pull/43).  :)

I considered making the quote text a child node, however historically you guys have expressed a preference for more opinionated components.  Putting the quote as a child node could allow the users to put any type of content they want there, and also leaves the styling of the quote text and credit text entirely up to the user.

Credit is the entity who the quote is credited to.  Footnotes aren't really proper in the context of quotes.